openassessment/xblock/job_sample_grader/code_grader.py

```diff
@@ -28,6 +28,17 @@ class TestGrader:
     __TMP_DATA_DIR__ = "/tmp/"
 
     def grade(self, response, add_staff_cases=False):
+        """
+        Setting prerequiste to execute code i,e. 
```

Comment:
It is not just setting pre-requisite. Code execution is happening in the same method.